src/nb_clean/__init__.py

```diff
@@ -82,6 +82,7 @@ def git_attributes_path() -> pathlib.Path:
 def add_git_filter(
     *,
     remove_empty_cells: bool = False,
+    remove_notebook_metadata: bool = False,
```

Comment:
We now have two arguments to this (and other functions), `remove_notebook_metadata` and `preserve_notebook_metadata`, which both default to `False` which is semantically inconsistent: we need to renconcile this in a way that maintains backward compatibility for existing users of `--preserve-notebook-metadata`.